web/frontend/src/pages/ballot/components/Select.tsx

```diff
@@ -102,25 +102,25 @@ const Select: FC<SelectProps> = ({ select, answers, setAnswers, language }) => {
       <div className="sm:pl-8 mt-2 pl-6">
         {Array.from(answers.SelectAnswers.get(select.ID).entries()).map(
           ([choiceIndex, isChecked]) => {
-            if (select.ChoicesMap.get('en') == undefined) return;
-            if (language === 'en' && select.ChoicesMap.has('en'))
+            if (language === 'fr' && select.ChoicesMap.has('fr'))
               return choiceDisplay(
                 isChecked,
-                select.ChoicesMap.get('en')[choiceIndex],
+                select.ChoicesMap.get('fr')[choiceIndex],
                 choiceIndex
               );
-            else if (language === 'fr' && select.ChoicesMap.has('fr'))
+            else if (language === 'de' && select.ChoicesMap.has('de'))
               return choiceDisplay(
                 isChecked,
-                select.ChoicesMap.get('fr')[choiceIndex],
+                select.ChoicesMap.get('de')[choiceIndex],
                 choiceIndex
               );
-            else if (language === 'de' && select.ChoicesMap.has('de'))
+            else if (select.ChoicesMap.has('en'))
               return choiceDisplay(
                 isChecked,
-                select.ChoicesMap.get('de')[choiceIndex],
+                select.ChoicesMap.get('en')[choiceIndex],
                 choiceIndex
               );
+            return '';
```

Comment:
Why not just `return` as it was the case before ?

Reply:
Because eslint complains that `map` wants a value to be returned.